pythonwhat/tasks.py

```diff
@@ -362,41 +362,73 @@ def taskRunEval(
     tempname="_evaluation_object_",
     call=None,
 ):
+    """
+    Eval an expression tree or node (with setting envs, pre_code and/or expr_code)
+    Utility function later wrapped to extract either result (end state of a variable), output (stdout) or error
+
+    Args:
+        tree (ast): current focused ast, used to get code to execute
+        process: manages shell (see locals.py)
+        shell: link to to get process namespace from execution up until now
+        env: variables to be replaced in focused code by name
```

Comment:
update value is more clear (replace could be text)

Reply:
Changed to "update value in focused code by name", correct?